frontend/src/resources/policy.test.ts

```diff
@@ -0,0 +1,66 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { Policy, PolicyTemplate, PolicySeverity, getPolicySeverity } from './policy'
+
+const testPolicy = <Policy>{
+  spec: {
+    disabled: false,
+    remediationAction: 'inform',
+  },
+}
+
+export function generateSeverity(severity: string): PolicyTemplate {
+  return <PolicyTemplate>{
+    objectDefinition: {
+      spec: {
+        severity,
+      },
+    },
+  }
+}
+
+describe('getPolicySeverity', () => {
+  it('should return high with no policy templates', async () => {
+    const policy = { ...testPolicy }
+    expect(getPolicySeverity(policy)).toEqual(PolicySeverity.High)
+  })
+  it('should return low', async () => {
+    const policy = { ...testPolicy }
+    policy.spec['policy-templates'] = [generateSeverity('low'), generateSeverity('low')]
+    expect(getPolicySeverity(policy)).toEqual(PolicySeverity.Low)
+  })
+  it('should return medium', async () => {
+    const policy = { ...testPolicy }
+    policy.spec['policy-templates'] = [generateSeverity('low'), generateSeverity('medium')]
+    expect(getPolicySeverity(policy)).toEqual(PolicySeverity.Medium)
+  })
+  it('should return high', async () => {
+    const policy = { ...testPolicy }
+    policy.spec['policy-templates'] = [generateSeverity('high'), generateSeverity('medium')]
+    expect(getPolicySeverity(policy)).toEqual(PolicySeverity.High)
+  })
+  it('should return critical', async () => {
+    const policy = { ...testPolicy }
+    policy.spec['policy-templates'] = [generateSeverity('critical'), generateSeverity('high')]
```

Comment:
What if we change the order? what is expected result? [generateSeverity('high'), generateSeverity('critial')]